.travis.yml

```diff
@@ -24,4 +24,21 @@ deploy:
     script: bash scripts/deploy_docker.sh $TRAVIS_TAG
     on:
       tags: true
-      branch: main
\ No newline at end of file
+      branch: main
+
+jobs:
```

Comment:
Currently I would include this in our normal build script (the `script` tag that already exists in the root of this file). So the `curl` and environment settings go before the `cargo build` and the `./grcov` goes after it.

We would only need multiple jobs if we want different compilers or os settings for different builds (which is not the case for us)

Alternatively we can remove the `script` tag from the root and inline the commands in the job `script`. The main change here is that we use `--verbose --all --locked` for `cargo test` (and `cargo build`)